task_templates/transforms/python3_image_transform/model-metadata.yaml

```diff
@@ -0,0 +1,35 @@
+name: python3_anomaly_detection
+type: training
+targetType: transform
+# modelID: optional
+environmentID: 5e8c889607389fe0f466c72d
```

Comment:
silly question.  what is this environmentID ?  is it a specific execution env in a specific instance of DR?

Reply:
environment IDs are defined in the dropen environment env_info file and is the same between environments